cocos/particle/animator/force-field.ts

```diff
@@ -235,7 +238,7 @@ export default class ForceField {
         const forceRandom = pseudoRandom(p.randomSeed + ForceRandSeed.FORCE);
         const forceMultiplier = multiplier.evaluate(normalizedTime, forceRandom);
 
-        Vec3.transformMat4(this._toForce, p.position, worldToLocal);
+        Vec3.transformMat4(this._toForce, _temp_local_pos, worldToLocal);
```

Comment:
Seems not correct. `_temp_local_pos` has not been initialized.

Reply:
_temp_local_pos has initialized before